### PR TITLE
Feature/177160902 fetch plottypes and options from db

### PR DESCRIPTION
**Pull request by upendrakumbham**
This PR was created only to merge [Previous PR](https://github.com/ebi-gene-expression-group/atlas-web-single-cell/pull/164) to the development branch. 

Previous PR was created from the master branch so I have to move all the changes manually to the new branch created from the develop branch. Once this new PR is approved then I merge to develop branch. So the all the changes available in the develop branch.

The previous PR was approved by @lingyun1010, so @lingyun1010 please have a glance at this PR to approve.

### CODE REVIEW

File: app/src/test/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentServiceIT.java [hunk 1]
```diff
@@ -67,7 +67,9 @@ void populateDatabaseTables() {
         populator.addScripts(
                 new ClassPathResource("fixtures/experiment-fixture.sql"),
                 new ClassPathResource("fixtures/scxa_tsne-fixture-alt.sql"),
-                new ClassPathResource("fixtures/scxa_cell_clusters-fixture-alt.sql"));
+                new ClassPathResource("fixtures/scxa_cell_clusters-fixture-alt.sql"),
+                new ClassPathResource("fixtures/scxa_cell_clusters-fixture-alt.sql"),
```
Comment (lingyun1010) [inline app/src/test/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentServiceIT.java:71]: Here is a duplicate one.

File: app/src/test/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentServiceIT.java [hunk 2]
```diff
@@ -77,7 +79,9 @@ void cleanDatabaseTables() {
         populator.addScripts(
                 new ClassPathResource("fixtures/experiment-delete.sql"),
                 new ClassPathResource("fixtures/scxa_tsne-delete.sql"),
-                new ClassPathResource("fixtures/scxa_cell_clusters-delete.sql"));
+                new ClassPathResource("fixtures/scxa_cell_clusters-delete.sql"),
+                new ClassPathResource("fixtures/scxa_cell_clusters-delete.sql"),
```
Comment (lingyun1010) [inline app/src/test/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentServiceIT.java:83]: Please remove the duplicate line.

File: app/src/test/java/uk/ac/ebi/atlas/experimentpage/TSnePlotSettingsServiceIT.java [hunk 3]
```diff
@@ -76,7 +78,9 @@ void cleanDatabaseTables() {
                 new ClassPathResource("fixtures/experiment-delete.sql"),
                 new ClassPathResource("fixtures/scxa_tsne-delete.sql"),
                 new ClassPathResource("fixtures/scxa_cell_clusters-delete.sql"),
-                new ClassPathResource("fixtures/scxa_analytics-delete.sql"));
+                new ClassPathResource("fixtures/scxa_analytics-delete.sql"),
+                new ClassPathResource("fixtures/scxa_analytics-delete.sql"),
```
Comment (lingyun1010) [inline app/src/test/java/uk/ac/ebi/atlas/experimentpage/TSnePlotSettingsServiceIT.java:82]: Duplicate line.

**Comment by pcm32** [comment]
If I understood correctly, this is a utility PR which is no longer is needed right? if that is the case, can we close it please? Thanks.

**Comment by upendrakumbham** [comment]
> If I understood correctly, this is a utility PR which is no longer needed, right? if that is the case, can we close it please? Thanks.

Done @pcm32. Thanks.

**Pull request closed without merging**